xml/art_amd-sev.xml

```diff
@@ -487,6 +491,12 @@ spec:
      &mdash; libvirt SEV configuration settings
     </para>
    </listitem>
+   <listitem>
+    <para>
+     <link xlink:href="https://libvirt.org/kbase/launch_security_sev.html"/>
+     &mdash; libvirt knowledge base article on AMD SEV
+    </para>
+   </listitem>
   </itemizedlist>
```

Comment:
should we mention also virt-manager configuration specifically for SUSE Linux 15SP4 as a TP Feature?
Explaining where to find the checkbox on launch for virt-manager (advanced dialog, memory section as "Launch Security"), and also mentioning the use of virt-install to install the guests, clarifying the additional launch security values/options? @jfehlig (and at charles, but could not find the user)

Reply:
well at least we should doc it in 15SP4 Release notes or doc, or even if we decide to mark it as TP, nobody will have a chance to even try the feature...

Reply:
@aginies this is possible something for at least 15SP4 Release notes, to sync with Charles (on 15SP4 we have some downstream ease-of-life option in virt-manager to enable launch security backed by AMD SEV. Probably we can mention this virt-manager enablement as TP or whatever, may be replaced by a slightly different GUI solution in the next releases)

Reply:
It could be added in the TP section of the RN.

Reply:
We can add it to the SEV doc too, but can be a separate PR. No need for it to block this PR.